### ISSUE TITLE
"panic: runtime error: index out of range" when running httpu new

### ISSUE THREAD

**Issue by issmirnov** [hazbo/httpu#10]
```
$ build/httpu new
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/hazbo/httpu/cmd/httpu/commands.newValue(0xc42000c060, 0x0, 0x0, 0x9f2d5801, 0xb36abab0961a5fa6)
	/Users/vania/go/src/github.com/hazbo/httpu/cmd/httpu/commands/new.go:24 +0x1d9
github.com/hazbo/httpu/cmd/httpu/commands.glob..func2(0xc42000c060, 0x0, 0x0, 0xc4200991e8, 0x1008cad)
	/Users/vania/go/src/github.com/hazbo/httpu/cmd/httpu/commands/new.go:58 +0x3f
github.com/hazbo/httpu/cmd/httpu/commands.(*Command).Run(0x1472c50, 0xc42000c060, 0x0, 0x0, 0xc420099268, 0xc42001e001)
	/Users/vania/go/src/github.com/hazbo/httpu/cmd/httpu/commands/commands.go:12 +0x48
main.main()
	/Users/vania/go/src/github.com/hazbo/httpu/cmd/httpu/main.go:60 +0xee
```

There is a bug at [cmd/httpu/commands/new.go#L24](https://github.com/hazbo/httpu/blob/master/cmd/httpu/commands/new.go#L24). I suggest modifying that block to check for the existence of args, and if there isn't one printing a help message.

```
newFlagSet.Parse(args)
if len(args) == 0 {
	newFlagSet.Usage() // Print proper usage. Or use fmt.Printf()
    os.Exit(1)
}
```

**Comment by AdvancedHacker101** [comment]
This is already fixed in my currently pending PR #9 .
Just FYI :)

**Comment by hazbo** [comment]
Thanks @issmirnov for the catch! Looks like @AdvancedHacker101 has picked this up, and has been merged 0a504f5349329d070e40becf831a193fb6632c38
